blocks/library/quote/index.js

```diff
@@ -74,12 +74,12 @@ registerBlock( 'core/quote', {
 		);
 	},
 
-	save( attributes ) {
+	save( { attributes } ) {
 		const { value, citation, style = 1 } = attributes;
 
 		return (
 			<blockquote className={ `blocks-quote-style-${ style }` }>
-				{ value && value.map( ( paragraph, i ) => (
+				{ value && wp.element.Children.map( value, ( paragraph, i ) => (
```

Comment:
Based on the `value` attributes definition, we should be certain that `value` is an array, not one of React's opaque children structure:

```js
query( 'blockquote > p', children() )
```